src/views/PeopleSearch/components/SearchFieldList/SearchFieldList.module.scss

```diff
@@ -0,0 +1,18 @@
+.people {
+  background-color: var(--mui-palette-neutral-light);
+  color: var(--mui-palette-primary-main);
+
+  &_searchbar {
+    background-color: var(--mui-palette-neutral-light);
+    color: var(--mui-palette-primary-main);
+  }
```

Comment:
`.people` and `.people_searchbar` are never used.

Reply:
Ah, I think this must have gotten missed, or messed up somehow in our merging.  I made some changes so all the classes are used, and every classname="" references a class that exists.